SprintDataset.py

```diff
@@ -331,6 +331,11 @@ def addNewData(self, features, targets=None, segmentName=None):
       assert "orth_classes" not in targets
       targets["orth_classes"] = numpy.array(self.orth_vocab.get_seq(orth), dtype="int32")
 
+    #build sequence end array
+    sequence_end = numpy.zeros((T,1), dtype="float32")
+    sequence_end[-1][0] = 1
+    targets["sequence_end"] = sequence_end
```

Comment:
In TF `EvalLayer`, sth like:
```
tf.equal(source(0, as_data=True).get_sequence_length(), tf.range(....))
```

Reply:
`tf.range` should be of shape (batch=1, time).
`...get_sequence_length()` is of shape (batch,)
Maybe needs `tf.expand_dims`.
Then you get the result of shape (batch,time).